enterprise/reporting/error_reporting.go

```diff
@@ -199,6 +217,10 @@ func (edr *ErrorDetailReporter) Report(ctx context.Context, metrics []*types.PUR
 
 		workspaceID := edr.configSubscriber.WorkspaceIDFromSource(metric.ConnectionDetails.SourceID)
 		metric := *metric
+		if !edr.IsPIIReportingDisabled(workspaceID) {
```

Comment:
Shouldn't it be the other way around? If it is disabled then don't report and return from here.
```suggestion
		if edr.IsPIIReportingDisabled(workspaceID) {
```